app/controllers/conversations_controller.rb

```diff
@@ -91,15 +91,20 @@ def new
       return
     end
 
-    @contacts_json = contacts_data.to_json
-    @contact_ids = ""
-
     if params[:contact_id]
-      @contact_ids = current_user.contacts.find(params[:contact_id]).id
+      gon.push conversation_prefill: [current_user.contacts.find(params[:contact_id]).person.as_json]
     elsif params[:aspect_id]
-      @contact_ids = current_user.aspects.find(params[:aspect_id]).contacts.map{|c| c.id}.join(',')
+      gon.push conversation_prefill: current_user.aspects.find(params[:aspect_id]).contacts.map {|c| c.person.as_json }
```

Comment:
Do we need these for the mobile view, too? Otherwise you could move this down into the `else` branch.